.ci/scripts/distribution/it-java.sh

```diff
@@ -5,6 +5,8 @@ export JAVA_TOOL_OPTIONS="$JAVA_TOOL_OPTIONS -XX:MaxRAMFraction=$((LIMITS_CPU))"
 
 tmpfile=$(mktemp)
 
+mvn -B -T$LIMITS_CPU -s ${MAVEN_SETTINGS_XML} test-compile -Pprepare-offline -pl qa/integration-tests -pl upgrade-tests
```

Comment:
Note that it's still upgrade-tests in 0.24

Reply:
Yeah the PR is still not merged